sequence/tests/component_test_dynamo_sequence.py

```diff
@@ -7,7 +7,7 @@
 from utilities import config
 from utilities import test_utilities
 
-from sequence.outbound import dynamo_sequence
```

Comment:
Possibly related to moving files around, `component_test_dynamo_sequence.py` no longer runs as part of the build because of missing __init__.py files

Reply:
__init__.py added